x/pool-incentives/keeper/keeper.go

```diff
@@ -58,19 +63,24 @@ func (k Keeper) Logger(ctx sdk.Context) log.Logger {
 	return ctx.Logger().With("module", "x/"+types.ModuleName)
 }
 
+// CreatePoolGauges checks if the poolId is CLPoolType, if it is create one gauge,
+// otherwise create multiple gauges based on lockableDurations.
 func (k Keeper) CreatePoolGauges(ctx sdk.Context, poolId uint64) error {
-	// Create the same number of gaugeges as there are LockableDurations
-	for _, lockableDuration := range k.GetLockableDurations(ctx) {
-		gaugeId, err := k.incentivesKeeper.CreateGauge(
+	pool, err := k.poolmanagerKeeper.RoutePool(ctx, poolId)
+	if err != nil {
+		return err
+	}
+	isCLPool := pool.GetType() == poolmanagertypes.Concentrated
+	if isCLPool {
+		gaugeIdCL, err := k.incentivesKeeper.CreateGauge(
 			ctx,
 			true,
 			k.accountKeeper.GetModuleAddress(types.ModuleName),
 			sdk.Coins{},
+			// dummy variable so that the existing logic doesnot break
 			lockuptypes.QueryCondition{
-				LockQueryType: lockuptypes.ByDuration,
-				Denom:         gammtypes.GetPoolShareDenom(poolId),
-				Duration:      lockableDuration,
-				Timestamp:     time.Time{},
+				LockQueryType: lockuptypes.ByTime,
```

Comment:
Would it be possible to leave `QueryCondition` fully blank? e.g. `lockuptypes.QueryCondition{}`

Reply:
unfortunately, i cannot, not without modifying `CreateGauge` function because it is default to Query `lockuptypes.ByDuration`

Reply:
Please add an issue to revisit and refactor `CreateGauge` and the hack here

Reply:
@stackman27 was there an issue created?

Reply:
yup forgot to tag it https://github.com/osmosis-labs/osmosis/issues/4833